### ISSUE TITLE
inline rendering svg 

### ISSUE THREAD

**Issue by t3ndai** [altair-viz/altair_saver#64]
for example to get an svg output in pygal, I would do this 
```
    chart = pygal.Line(disable_xml_declaration=True, x_label_rotation=45)
    return chart.render(is_unicode=True)
```
what's the equivalent in altair so I can get an svg embedded string, not a *file*  that I can easily save to the db for output later ?

**Comment by cheak1974** [comment]
Very good question. I was about asking the same. I use PyGal and load the rendered svg output as embedded image to the background of div tags in my Web UI. There should be no need to write the image data to disk when you could keep it in memory instead.

cheers

**Comment by jakevdp** [comment]
This will return an svg string:
```python
import altair_saver
altair_saver.save(chart, fmt='svg')
```

**Comment by joelostblom** [comment]
Since Altair 5.2, the functionality of Altair Saver is now available in Altair via the [vl-convert](https://github.com/vega/vl-convert) package. Most of the functionality has been available since 5.0, and [the main addition in 5.2 was PDF export](https://github.com/altair-viz/altair/pull/3244). See [the docs on how to save charts](https://altair-viz.github.io/user_guide/saving_charts.html#png-svg-and-pdf-format) for more details.

We are going to archive this repo, so I'm closing all the open issues and PRs before doing so. Try out the new options for saving charts mentioned above and if you run into issues, please open an issue directly in the altair or vl-convert repo.
